lib/src/generated/bledata.pbjson.dart

```diff
@@ -195,16 +368,44 @@ const DiscoverServicesInfo$json = const {
   '1': 'DiscoverServicesInfo',
   '2': const [
     const {'1': 'deviceId', '3': 1, '4': 1, '5': 9, '10': 'deviceId'},
-    const {'1': 'services', '3': 2, '4': 3, '5': 11, '6': '.DiscoveredService', '10': 'services'},
+    const {
+      '1': 'services',
+      '3': 2,
+      '4': 3,
+      '5': 11,
+      '6': '.DiscoveredService',
+      '10': 'services'
+    },
   ],
 };
 
 const DiscoveredService$json = const {
   '1': 'DiscoveredService',
   '2': const [
-    const {'1': 'serviceUuid', '3': 1, '4': 1, '5': 11, '6': '.Uuid', '10': 'serviceUuid'},
-    const {'1': 'characteristicUuids', '3': 2, '4': 3, '5': 11, '6': '.Uuid', '10': 'characteristicUuids'},
-    const {'1': 'includedServices', '3': 3, '4': 3, '5': 11, '6': '.DiscoveredService', '10': 'includedServices'},
+    const {
+      '1': 'serviceUuid',
+      '3': 1,
+      '4': 1,
+      '5': 11,
+      '6': '.Uuid',
+      '10': 'serviceUuid'
+    },
+    const {
+      '1': 'characteristicUuids',
+      '3': 2,
+      '4': 3,
+      '5': 11,
+      '6': '.Uuid',
+      '10': 'characteristicUuids'
+    },
+    const {
+      '1': 'includedServices',
+      '3': 3,
+      '4': 3,
+      '5': 11,
+      '6': '.DiscoveredService',
+      '10': 'includedServices'
+    },
```

Comment:
I kind of hate Protobuf. But I love that it synchronizes multiple parties (languages) together, so it's a love-hate relationship 🤔

Reply:
same here :-)